src/main/scala/chiseltest/stage/ChiselTestAnnotations.scala

```diff
@@ -122,6 +122,14 @@ case object VcsBackendAnnotation extends BackendAnnotation {
   )
 }
 
+case class TestCommandAnnotation(commands: Seq[String]) extends NoTargetAnnotation with ChiselTestOption
```

Comment:
Short scaladoc for these would be nice.

Reply:
will fix.

Reply:
Documentation with correspond bug fixes.